.github/workflows/releases.yml

```diff
@@ -60,31 +71,7 @@ jobs:
           .\script\sign.ps1 -Certificate "${{ steps.obtain_cert.outputs.cert-file }}" `
             -Executable "${{ steps.buildmsi.outputs.msi }}"
       - name: Upload MSI
-        uses: ./.github/actions/upload-msi
-        with:
-          msi-file: ${{ steps.buildmsi.outputs.msi }}
+        shell: bash
+        run: bin/hub release edit "${GITHUB_REF#refs/tags/}" -m "" -a "${{ steps.buildmsi.outputs.msi }}"
```

Comment:
Since the upload MSI step only needs to upload a single file and is no longer responsible for deleting the old `.exe`, I've chosen to just do a hub one-liner here to replace the JS action.